composeApp/src/commonMain/composeResources/values/strings.xml

```diff
@@ -1,2 +1,22 @@
 <resources>
+    <string name="app_name">Minesweeper</string>
+
+    <string name="apply">Apply</string>
+    <string name="cancel">Cancel</string>
+    <string name="mine_count">Mine count</string>
+    <string name="width">Width</string>
+    <string name="height">Height</string>
+    <string name="restart">Restart</string>
+    <string name="settings">Settings</string>
+
+    // TODO: Check how Accessibility work, if its valueble - change trailing for: %d mines left.
+    // TODO: Accessibility check gives info that user can't select @Composable Counter`s with tab-navigation in our app - need to undertsand why and fix it.
+    <string name="mines_counter">Counter of remaining mines, mines left: %d</string>
```

Comment:
Since this is for accessibility, maybe find a shorter text? E.g. "Mines left: %d".

Reply:
Also, let's add a prefix for all accessibility strings, e.g. `desc_mines_left`.